wdl-format/tests/format/ENCODE-DCC_chip-seq-pipeline/source.formatted.wdl

```diff
@@ -1511,38 +1380,26 @@ workflow chip {
     scatter (i in range(num_rep)) {
         # to override endedness definition for individual replicate
         #     paired_end will override paired_ends[i]
-        Boolean paired_end_ = (
-            if !defined(paired_end) && i < length(paired_ends)
-            then paired_ends[i]
-            else select_first([
-                paired_end,
-            ])
-        )
+        Boolean paired_end_ = if !defined(paired_end) && i < length(paired_ends) then paired_ends[
+            i] else select_first([
```

Comment:
I'm on the fence whether I like this idea that `(` and `[` can split things down to the next line. For instance, here, I think it looks quite strange. I'm thinking that, for these lines that are being broken based on long line length, perhaps that should not be permitted.

Reply:
I'm planning for a follow up PR that will line break "matching" tokens. So this example might become:
```
        Boolean paired_end_ = if !defined(paired_end) && i < length(paired_ends) then paired_ends[
            i
        ] else select_first([
```
or something similar. I haven't spent time thinking about the specifics yet, but I agree that the current implementation leaves something to be desired. I can push up a change the removes the open/close tokens from `can_be_line_broken` so we can see how it looks?